src/playground/components/AlertsActionBar.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+export default function AlertsActionBar({
+	messages,
+	onFixAll,
+	options,
+	onUpdate,
+	rulesWithInvalidConfigs,
+	setRulesWithInvalidConfigs,
+}) {
+	return (
+		<div className="alerts-action-bar">
+			<span className="alerts-action-bar__description">
+				Resolve all fixable issues or disable their respective rules
+				from config
```

Comment:
Since now playground showing the `Fix All` and `Disable All` together or only `Disable All` accordingly can we also show this description accordingly?